### PR TITLE
Enable CD

### PR DESCRIPTION
**Pull request by jglick**
https://github.com/jenkinsci/reverse-proxy-auth-plugin/pull/153#issuecomment-2479266363 after https://github.com/jenkins-infra/repository-permissions-updater/pull/4177

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: monthly
+# the rest handled by Renovate
```
Comment (timja) [inline .github/dependabot.yml:7]: bit confused, renovate will also do github actions...

Comment (jglick) [reply .github/dependabot.yml:7]: Is it already configured to do so? If so, I can just delete the file. I am not familiar with what https://github.com/jenkinsci/reverse-proxy-auth-plugin/blob/be58e138d08eaf2b590f2266c01ff1012363ed22/.github/renovate.json#L4 would include.

Comment (timja) [reply .github/dependabot.yml:7]: some links:
https://docs.renovatebot.com/presets-config/#configrecommended

https://docs.renovatebot.com/presets-group/#grouprecommended

its hard to tell from just looking there but AFAIK all languages are enabled by that, e.g. see Jenkins core:

https://github.com/jenkinsci/jenkins/pull/9963

https://github.com/jenkinsci/jenkins/blob/master/.github/renovate.json#L4

**Comment by jglick** [comment]
(Still waiting for https://github.com/jenkinsci/reverse-proxy-auth-plugin/settings/secrets/actions to show the Artifactory token.)

**Review by sboardwell — APPROVED** [review]

**Pull request merged**